test/snippet/core/detail/strong_type_usage.cpp

```diff
@@ -0,0 +1,29 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/std/ranges>
+
+
+namespace seqan3::detail
+{
+template <std::ranges::ForwardRange fwd_rng_type>
+    bool do_find(fwd_rng_type &, int const, int const) { return true; }
```

Comment:
```suggestion
bool do_find(fwd_rng_type &, int const, int const) { return true; }
```

Here again: why is a seqan namespace opened in a snippet?